tutorials/R/README.md

```diff
@@ -381,7 +371,9 @@ Best parameters :
 
 max_depth | margin |min_sample | loss |n_trees |n_features|
 | ---- |:------: | ---------:|---: |---: |---: |
-3         |5       |          2|  hinge|      10|          2|
+3         |2       |          5|  square |      10|          10|
+
+*<b> Warning </b> : The model and predictions may vary from the example due to random sampling*
```

Comment:
also it would be better to put this tutorial in the standard R pkg location, e.g. mmit/Rpackage/vignettes/tutorial.Rmd

see http://kbroman.org/pkg_primer/pages/vignettes.html for more info about how to write vignettes. it is ok to leave these warnings as is, but please open a separate PR for moving this tutorial to a vignette.

Reply:
ok....and should I put the seed in random forest code in this PR and update the doc or open new PR for that?

Reply:
please keep this PR focused on the stated purpose (changing parallel for loops to future.apply) and open a new PR for the random seed/vignette

Reply:
ok